src/cmd/services/m3aggregator/config/server.go

```diff
@@ -38,6 +39,9 @@ type RawTCPServerConfiguration struct {
 	// Raw TCP server listening address.
 	ListenAddress string `yaml:"listenAddress" validate:"nonzero"`
 
+	// Compress options.
+	Compress client.CompressType `yaml:"compress"`
+
```

Comment:
nit: similar case re: naming here, `compression` over `compress`? Also comments in this file don't seem to have correct formatting, but happy to punt on that